<launch>
	<!-- Define the remote Jetson machine to launch nodes -->
	<!-- TODO: Do we want to launch the Jetson this way, or as a stand-alone - this way at least ensures that roscore is running -->
	<!-- machine name="num-sb1-jetson" address="num-sb1-jetson" env-loader="/opt/numurus/ros/setup.bash" user="nvidia" -->
	
	<!-- Everything lives in the "numurus" namespace -->
	<group ns="numurus">
		<!-- Load the base config file -->
		<rosparam command="load" file="/opt/numurus/ros/etc/sb1.yaml" />

		<!-- Local nodes -->
		<node name="timestamp_mgr" pkg="num_sdk_zynq" type="timestamp_mgr_node" respawn="true" output="log" />
		<node name="trigger_mgr" pkg="num_sdk_zynq" type="trigger_mgr_node" respawn="true" output="log" />
		<node name="time_sync_mgr" pkg="num_sdk_zynq" type="time_sync_mgr.py" respawn="true" output="log" />
		<node name="test_pat_generator" pkg="num_sdk_zynq" type="test_pat_generator_node" respawn="true" output="log" />
	</group>
</launch>